plugin/pkg/scheduler/generic_scheduler_test.go

```diff
@@ -397,6 +398,23 @@ func makeNode(node string, milliCPU, memory int64) *v1.Node {
 	}
 }
 
+func TestHumanReadableFitError(t *testing.T) {
+	error := &FitError{
+		Pod: &v1.Pod{ObjectMeta: v1.ObjectMeta{Name: "2"}},
+		FailedPredicates: FailedPredicateMap{
+			"1": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeUnderMemoryPressure},
+			"2": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeUnderDiskPressure},
+			"3": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeUnderDiskPressure},
+		},
+	}
+	if strings.Contains(error.Error(), "No nodes are available that match all of the following predicates") {
```

Comment:
Ah yes, this is 
- to avoid Overtesting: We shouldn't be testing the order in which the predicate failures are listed, as thats neither a requirement nor a gauranteed quality of maps in golang: https://nathanleclaire.com/blog/2014/04/27/a-surprising-feature-of-golang-that-colored-me-impressed/.

We could print it multiple times and gaurantee that the exact string is eventually provided, but that would be awkward.